services/movie_session.py

```diff
@@ -26,11 +27,11 @@
 
 def update_movie_session(
     session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+    show_time: Optional[str] = None,
+    movie_id: Optional[int] = None,
+    cinema_hall_id: Optional[int] = None,
 ) -> None:
-    movie_session = MovieSession.objects.get(id=session_id)
+    movie_session: MovieSession = MovieSession.objects.get(id=session_id)
     if show_time:
         movie_session.show_time = show_time
```

Comment:
The `show_time` parameter is expected to be a string, but it should be a `datetime` object to match the model's field type. Ensure that the input is properly converted before saving.